api4/user.go

```diff
@@ -1622,23 +1622,24 @@ func enableUserAccessToken(c *Context, w http.ResponseWriter, r *http.Request) {
 	ReturnStatusOK(w)
 }
 
-func registerServiceTermsAction(c *Context, w http.ResponseWriter, r *http.Request) {
+func registerTermsOfServiceAction(c *Context, w http.ResponseWriter, r *http.Request) {
 	props := model.StringInterfaceFromJson(r.Body)
 
 	userId := c.Session.UserId
-	serviceTermsId := props["serviceTermsId"].(string)
+	// TODO refactor this to be terms of service
```

Comment:
@harshilsharma: TODO here re: discussion.

Reply:
Ah yes, the TODOs have already been implemented. Removing them.